pureStorageDriver/README.md

```diff
@@ -7,6 +7,8 @@ This helm chart installs the Pure Service Orchestrator CSI plugin on a Kubernete
 1. Currently, there is **no upgrade supported** from previous versions that do not deploy the datastore.
 1. You **MUST** supply a unique `clusterID` in values.yaml. This was previously called `namespace.pure`. `clusterID` must be less than or equal to 22 characters in length. `clusterID` must be unique between **all** Kubernetes clusters using your Pure devices or naming conflicts will result.
 1. `helm uninstall` will perform the initial uninstallation, but some pods will continue to clean up post-installation. They should go away after cleanup is complete.
+1. Note that PSO CSI has upgraded its sidecars to support BETA version snapshotter APIs. The snapshotter CRDs for BETA version APIs have been upgraded, make sure for 1.17 and later versions, apply release-2.0 CRDs as shown
```

Comment:
[nit] I feel like we don't need to make *every* occurrence of "beta" all uppercase 😂  probably just the first one is fine.

Reply:
sounds good, thx